ramses_rf/devices_hvac.py

```diff
@@ -251,44 +251,102 @@ class HvacVentilator(DeviceHvac):  # FAN: RP/31DA, I/31D[9A]
+        return self._msg_value(_31DA, key=EXHAUST_TEMPERATURE)
+
+    @property
+    def supply_temperature(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=SUPPLY_TEMPERATURE)
+        
+    @property
+    def indoor_temperature(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=INDOOR_TEMPERATURE)
+        
+    @property
+    def outdoor_temperature(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=OUTDOOR_TEMPERATURE)
+
+    @property
+    def speed_cap(self) -> Optional[int]:
+        return self._msg_value(_31DA, key=SPEED_CAP)
+    
+    @property
+    def bypass_pos(self) -> Optional[int]:
+        return self._msg_value(_31DA, key=BYPASS_POS)
+    
+    @property
+    def fan_info(self) -> Optional[str]:
+        return self._msg_value(_31DA, key=FAN_INFO)
+    
+    @property
+    def supply_fan_speed(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=SUPPLY_FAN_SPEED)
+    
+    @property
+    def post_heat(self) -> Optional[int]:
+        return self._msg_value(_31DA, key=POST_HEAT)
+    
+    @property
+    def pre_heat(self) -> Optional[int]:
+        return self._msg_value(_31DA, key=PRE_HEAT)
+    
+    @property
+    def supply_flow(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=SUPPLY_FLOW)
+    
+    @property
+    def exhaust_flow(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=EXHAUST_FLOW)
 
     @property
     def status(self) -> dict:
         return {
             **super().status,
-            "exhaust_fan_speed": self.fan_rate,
-            **(
-                {
-                    k: v
-                    for k, v in self._msgs[_31D9].payload.items()
-                    if k != "exhaust_fan_speed"
-                }
-                if _31D9 in self._msgs
-                else {}
-            ),
-            **(
-                {
-                    k: v
-                    for k, v in self._msgs[_31DA].payload.items()
-                    if k != "exhaust_fan_speed"
-                }
-                if _31DA in self._msgs
-                else {}
-            ),
+            EXHAUST_FAN_SPEED: self.exhaust_fan_speed,
+            REMAINING_TIME: self.remaining_time,
+            CO2_LEVEL: self.co2_level,
+            INDOOR_HUMIDITY: self.indoor_humidity,
+            AIR_QUALITY: self.air_quality,
+            AIR_QUALITY_BASE: self.air_quality_base,
+            OUTDOOR_HUMIDITY: self.outdoor_humidity,
+            EXHAUST_TEMPERATURE: self.exhaust_temperature,
+            SUPPLY_TEMPERATURE: self.supply_temperature,
+            SPEED_CAP: self.speed_cap,
+            BYPASS_POS: self.bypass_pos,
+            FAN_INFO: self.fan_info,
+            SUPPLY_FAN_SPEED: self.supply_fan_speed,
+            POST_HEAT: self.post_heat,
+            PRE_HEAT: self.pre_heat,
+            SUPPLY_FLOW: self.supply_flow,
+            EXHAUST_FLOW: self.exhaust_flow,
```

Comment:
I think create a constant tuple in this module:
```python
STATUS_ATTRS = (
    AIR_QUALITY,
    AIR_QUALITY_BASE
    ...
  )
  ```
  ... then we can use a dict comprehension:
  ```python
      **{k, getattr(self, k) for k in STATUS_ATTRS}
  ```

Reply:
Not sure if I did this correctly, please have a look